lib/location/scan_wifi.c

```diff
@@ -59,6 +59,44 @@ void scan_wifi_details_get(struct location_data_details *details)
 }
 #endif
 
+#if defined(LOCATION_METHOD_WIFI_NET_IF_UPDOWN)
```

Comment:
```suggestion
#if defined(CONFIG_LOCATION_METHOD_WIFI_NET_IF_UPDOWN)
```

Affects all occurences.